framework/include/outputs/Exodus.h

```diff
@@ -180,12 +190,14 @@ class Exodus : public OversampleOutput
   /// Storage for input file record; this is written to the file only after it has been initialized
   std::vector<std::string> _input_record;
 
+protected:
```

Comment:
I have been guilty of this myself in the past, but let's try really hard to avoid interweaving access levels. Can we move the protected items ahead in the initializer list?

Reply:
Should be done now.